modules/transactionpool/transactionpool.go

```diff
@@ -102,9 +104,12 @@ func New(cs modules.ConsensusSet, g modules.Gateway, persistDir string) (*Transa
 	if err != nil {
 		return nil, err
 	}
+	if build.DEBUG && build.Release != "testing" {
+		fmt.Println("\tTransaction Pool: persist initialization complete")
+	}
 
 	// Register RPCs
-	g.RegisterRPC("RelayTransactionSet", tp.relayTransactionSet)
+	go g.RegisterRPC("RelayTransactionSet", tp.relayTransactionSet)
```

Comment:
transaction pool sometimes hangs at startup, felt like it might make a difference.